.release-please-manifest.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
I added this when things weren't working after the initial addition of release-please, but I see it's not required, see https://github.com/netlify/vite-plugin-netlify-edge.